lambdas/build-user-identity/src/test/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandlerTest.java

```diff
@@ -246,6 +248,53 @@ void shouldReturnCredentialsWithCiMitVCOnSuccessfulUserInfoRequest()
         verify(mockCiMitService, times(1)).getContraIndicatorsVCJwt(any(), any(), any());
     }
 
+    @Test
+    void
+            shouldReturnCredentialsWithCiMitVCOnSuccessfulUserInfoRequestWhenBundleCimitVCFeatureFlagIsNotEnable()
```

Comment:
`shouldReturnCredentialsWithCiMitVCOnSuccessfulUserInfoRequestWhenBundleCimitVCFeatureFlagIsNotEnable` should be `shouldReturnCredentialsWithoutCiMitVCOnSuccessfulUserInfoRequestWhenBundleCimitVCFeatureFlagIsNotEnabled`?

Reply:
Done